raiden/ui/startup.py

```diff
@@ -134,7 +135,11 @@ def __post_init__(self) -> None:
             )
 
 
-def load_deployed_contracts_data(config: RaidenConfig, chain_id: ChainID) -> Dict[str, Any]:
+def load_deployed_contracts_data(
+    config: RaidenConfig,
+    chain_id: ChainID,
+    development_environment: ContractDevEnvironment = ContractDevEnvironment.DEMO,
```

Comment:
Is it missing a `.value`, or does it really expects an enum type?

Reply:
Using the enum keeps the type checking, allows pretty printing and querying the name. Why should I use the value unless I'm forced to? Isn't this how python enums are usually used?

Reply:
It depends on what the function is expecting and how it's using this parameter.
But ok, this already answered my question, I just wanted to make sure this wasn't a mistake.